druid.go

```diff
@@ -185,6 +201,12 @@ func WithBasicAuth(username, password string) ClientOption {
 	}
 }
 
+func WithSkipTLSVerify(skip bool) ClientOption {
```

Comment:
maybe the bool is useless here ? `WithSkipTLSVerify()` would be enough no ?

Reply:
Yes, I did think of that. If you are OK with that, I'll gladly include that change.

Reply:
Sure, let's do